templates/.rubocop.yml

```diff
@@ -23,3 +23,7 @@ AllCops:
     - "config/**/*"
     - "Rakefile"
     - "spec/i18n_spec.rb"
+
+BlockLength:
+  Exclude:
+    - "spec/**/*"
```

Comment:
Throw `app/admin` in there too.. even though not all projects use it, doesn't hurt to have it